.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+Linked PRs: (links to corresponding PRs, optional)
+
+## Changes
+- (changes here)
+
+## TODOs:
+- [ ] +1
+- [ ] Updated README
+- [ ] Updated CHANGELOG
+- [ ] (Other applicable TODOs)
```

Comment:
Since this is the PR template for _this_ repo... if there are other applicable TODOs shouldn't we just list them here? Or is the intent to have the PR author add other things they might want to make sure they don't forget in this list on a per-PR basis?